### DISCUSSION TITLE
A-Z quickselect?

### DISCUSSION THREAD

**Discussion by timothybest** [advplyr/audiobookshelf#3101] [Ideas]
I love audiobookshelf and thanks to everyone for all the hard work.  I would like to suggest an A-Z Quickselect option on the right side of the Library screen to move quickly through the booklist - I am sure you are familiar with the one that Plex has - something like that would be very helpful.

**Comment by nichwall** [comment]
https://github.com/advplyr/audiobookshelf/issues/2165
https://github.com/advplyr/audiobookshelf-app/issues/544
